iptables-detect/iptables-detect.sh

```diff
@@ -144,6 +144,13 @@ os_detect() {
 
     case "$lsb_dist" in
 
+    alpine)
```

Comment:
Since the same is done for "ubuntu" can we update the case below to be "ubuntu|alpine"?

Reply:
We can, but for now I'm trying to keep them separate in the event that in the future one of the distributions (most likely Ubuntu) switches to NFT as a standard, making the future diff a lot cleaner.